### ISSUE TITLE
Proposed SIG-Docs meeting agenda for 2024-08-14

### ISSUE THREAD

**Issue by ShaunaGordon** [o3de/sig-docs-community#128]
## Meeting Details

* **Date/Time:** Aug 14, 2024 @ 11:00 CDT/12:00 EDT/17:00 CEST (postponed 30 minutes due to Connect)
* **Location:** [Discord SIG-Docs Voice Room](https://discord.gg/p3padwr58u)
* **Moderator:** @RoddieKieley 
* **Note Taker:** @ShaunaGordon 

The [SIG-Docs Meetings](https://github.com/o3de/sig-docs-community/blob/main/meetings/README.md) contains the history past calls, including a link to the agenda, recording, notes, and resources.

## SIG Updates

**What happened since the last meeting?** - The creator of this issue is responsible for providing a brief summary here to be used as an update to start any important discussions.

### Open action items

* [ ] Waiting on sync PR to be merged
* [x] Shauna still waiting to hear back from Kythera
* [ ] Shauna still working on getting maintainer privs
* [ ] PRs triaged, just waiting merging or feedback from maintainers
* [ ] Start stabilization process
* [ ] Triage issues

### Newly resolved items

* :white_check_mark: Kythera responded, giving the greenlight to pull the content and requested we link to their Youtube channel (Shauna offered linking to them)
* ✅ Cherry-picked PR from main to dev completed and reviewed, waiting for merge

## Initial Agenda

The pre-approved agenda for this meeting is as follows:

* **Kythera** - **Shauna** - **1 min**
  * From Kythera: > I've confirmed with our CEO and Lead Architect that we have the pages on the Wayback Machine, so he's happy to say they are safe. As for a link to our tutorials, please could you add [a link to our YouTube channel](https://www.youtube.com/@kytheraai3330) which contains tutorials and explainers for a number of our tools?
* **Shauna's maintainership update** - **Shauna** - **5 min**
  * Some movement has been made. Might get perms soon.

## Suggesting and voting on agenda items

Our community is welcome to suggest and vote on agenda items. If you suggest an item you need to have a designated presenter (normally the issue filer), but any community member can vote and isn't required to attend meetings or even participate in further discussion. If you think something is a good idea to discuss for the O3DE community, please upvote!

:+1: reactions to proposed agenda items are counted as YEA votes for taking up discussion, and :-1: as NAY. Votes on items don't determine the final agenda, but do influence it and are strongly taken into consideration as a representation of what the O3DE community wants to see from the project.

### Agenda item format

If you leave a comment on this issue to propose an agenda item, please use the following format:

```
**Topic**: The topic you'll be presenting on
**Presenter**: Who will present (may be a team name or TBD)
**Length of time**: Estimate the amount of time you believe discussion will take. This should not be more than 15 minutes without SIG approval.

Include a description of your topic. Make sure to link to any supplementary materials such as RFCs, forum discussions, issues, etc.
```

Remember to add the :+1: and :-1: reactions manually, so they appear and are obvious to voters!
